packages/client/modules/userDashboard/components/OrgBilling/PaymentDetails.tsx

```diff
@@ -127,29 +124,16 @@ const PaymentDetails = (props: Props) => {
 
   useEffect(() => {
     if (tier !== 'starter') return
-    const handleCompleted: CompletedHandler<TCreateSetupIntentMutation['response']> = (res) => {
-      const {createSetupIntent} = res
-      const {clientSecret} = createSetupIntent
-      if (clientSecret) {
-        setClientSecret(clientSecret)
-      }
-    }
-    CreateSetupIntentMutation(atmosphere, {orgId}, {onError, onCompleted: handleCompleted})
+    CreateSetupIntentMutation(atmosphere, {orgId}, {onError, onCompleted})
   }, [])
 
-  if (!clientSecret.length) return null
   return (
     <StyledPanel label='Credit Card'>
       <StyledRow>
         <Plan>
           <Title>{'Credit Card Details'}</Title>
           <Content>
-            <Elements
-              options={{
-                clientSecret
```

Comment:
`clientSecret` was is no longer needed with `CardElement` and `confirmCardPayment`. This means we no longer need to return null before we get the `clientSecret` which fixes the jumpiness